src/etcd_client.rs

```diff
@@ -46,6 +47,21 @@ impl EtcdClient {
         Ok(result_vec)
     }
 
+    /// Get key-value from etcd
+    async fn get_kv_async<T: DeserializeOwned>(&self, key: &str) -> anyhow::Result<Option<T>> {
```

Comment:
how `get_kv_async` make sure retrieve only one kv pair?

Reply:
Change name to get_one_kv_async()

Reply:
Done